### ISSUE TITLE
Vaccination Data

### ISSUE THREAD

**Issue by dprensha** [CSSEGISandData/COVID-19#3390]
Once the COVID-19 vaccine is rolled out, does JHU intend to track vaccination data as well? It would be an extremely helpful metric to track, I think.

**Comment by StateCenterKid** [comment]
I agree that this would be a great add to this repository.   Has anyone seen indications that other sources will begin tracking vaccination counts?

**Comment by dprensha** [comment]
Any update on this?

**Comment by dprensha** [comment]
Apparently that data is [here](https://github.com/govex/COVID-19/tree/master/data_tables/vaccine_data), but it would be nice to have it consolidated with the JHU data set
